java/core/src/test/org/apache/orc/TestRowFilteringIOSkip.java

```diff
@@ -570,6 +570,29 @@ public void schemaEvolutionLong2StringColumn() throws IOException {
     assertEquals(1, rowCount);
   }
 
+  @Test
+  public void readCaseInsensitive() throws IOException {
```

Comment:
Do we have a case-sensitive test coverage?

Reply:
The default value is true for case-sensitivity so all the tests are the other tests are case-sensitive tests. I added an explicit failure test with case-sensitivity when the name is not found.

Reply:
Thank you so much!